GameServer/Logic/GameObjects/Champion.cs

```diff
@@ -82,23 +68,23 @@ public Champion(string type, uint playerId, RuneCollection runeList, uint netId
             stats.ManaRegenerationPerLevel = inibin.getFloatValue("DATA", "MPRegenPerLevel");
             stats.GrowthAttackSpeed = inibin.getFloatValue("DATA", "AttackSpeedPerLevel");
 
-            spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell1"), 0));
-            spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell2"), 1));
-            spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell3"), 2));
-            spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell4"), 3));
-            spells.Add(new Spell(this, "SummonerHeal", 4));
-            spells.Add(new Spell(this, "SummonerFlash", 5));
-            spells.Add(new Spell(this, "Recall", 13));
+            Spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell1"), 0));
+            Spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell2"), 1));
+            Spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell3"), 2));
+            Spells.Add(new Spell(this, inibin.getStringValue("Data", "Spell4"), 3));
+            Spells.Add(new Spell(this, "SummonerHeal", 4));
+            Spells.Add(new Spell(this, "SummonerFlash", 5));
+            Spells.Add(new Spell(this, "Recall", 13));
 
             setMelee(inibin.getBoolValue("DATA", "IsMelee"));
             setCollisionRadius(inibin.getIntValue("DATA", "PathfindingCollisionRadius"));
 
             Inibin autoAttack;
-            if (!_rafManager.readInibin("DATA/Characters/" + type + "/Spells/" + type + "BasicAttack.inibin", out autoAttack))
+            if (!_rafManager.readInibin("DATA/Characters/" + Model + "/Spells/" + Model + "BasicAttack.inibin", out autoAttack))
```

Comment:
I think we have a function in the RafManager for finding the autoattack inibin, this should be made to use that function.